core/src/test/scala/cromwell/core/path/PathBuilderSpecUtils.scala

```diff
@@ -87,16 +83,6 @@ trait PathBuilderSpecUtils {
         Option(path.getFileName).map(_.pathAsString).orNull should be(goodPath.getFileName)
       }
 
-    it should "match expected start of toUri" in
-      withClue(s"for path ${goodPath.path}${if (goodPath.normalize) " (normalized)" else ""}:") {
-        path.toUri.toString should startWith(goodPath.toUriStartsWith)
```

Comment:
Can you also comment out `NioPathMethods.toUri`, leaving a comment with a link to the existing Google NIO issue?

I don't want anyone tripping over this in the future.